README.md

```diff
@@ -35,33 +45,39 @@ effective to guide PR discussions.
 - [Contributing & Good Practices](#contributing--good-practices): recommendation on how to write good code for this
   application;
 - [License](#license): how this software is licensed and how you may use it.
+- [Sponsors](#sponsors): who helped us to make this project a reality.
 
 ## Setup
 
-If you have no idea how to install Flutter and run it locally, check this
-[_Get started_](https://flutter.dev/docs/get-started/install).
-
-If you have Flutter setup locally, on the project's root folder, install pubspec dependencies by running
-`flutter pub get`.
+<!-- TODO(matuella): Add links referencing such setups after both are done -->
 
-### Firebase dependencies
-
-Memo has Firebase dependencies that needs to be setup before running locally. More information on this in 
-[ARCHITECTURE#firebase](ARCHITECTURE.md#firebase).
+It shouldn't be hard to run your own Memo setup locally, but you'll have to add Firebase in both [`firebase/`]() and
+[`flutter/`]() projects. These are really simple steps, but the TLDR is that you'll have to create a Firebase project;
+deploy the infrastructure specified in `firebase/`; and add the required Firebase files to run your own server/client
+instances.
 
 ## Architecture
 
-How this application works from inside and how it interacts with external dependencies - written in details in
-[ARCHITECTURE.md](ARCHITECTURE.md).
+The root [ARCHITECTURE.md](ARCHITECTURE.md) gives an overview of this application's architecture, although if you want
+to know how Memo works from the inside, go check out both [Flutter's ARCHITECTURE](flutter/ARCHITECTURE.md) and
+[Firebase's ARCHITECTURE](firebase/ARCHITECTURE.md) - these go through all of the nitty-gritty of each language toolkit,
+in-application layers (and its interactions), non-standard decisions, etcetera - literally explaining every major
+decision for each "ecosystem".
 
 ## Background
 
-If you're interested in checking out an overview about how we dealt with this project's software process (inside our 
-team), check out [.process/](.process/README.md) (sorry, for now only in ptBR).
+If you're interested in checking out an overview - of the first release, client-side only - about how we dealt with this
+project's software process, check out [.process/](.process/README.md) (sorry, only in ptBR).
 
 ## Contributing & Good Practices
 
-See [CONTRIBUTING](CONTRIBUTING.md) for details about how to contribute to the project.
+Memo is all about collections (the word that we are using for our flashcards), and it means **a lot of content**. The
```

Comment:
```suggestion
Memo is all about collections (the word that we are using for our set of flashcards), and it means **a lot of content**. The
```

Reply:
I mean, the flashcards are Memos, and the collections are a set of Memos (flashcards)